### ISSUE TITLE
Investigate AVS operation on various operating systems

### ISSUE THREAD

**Issue by mbanth** [xmos/vocalfusion-avs-setup#75]
Currently, XMOS maintains this repository to allow a customer to set up AVS on a Raspberry Pi in order to evaulate our VocalFusion products.  We would like to enable customers to evaluate those products by running AVS on other platforms, e.g. Linux, MacOS or Windows.  If possible, this extended support will simplfy the evaluation set up because the customer can simply plug in a UA product variant to their laptop via USB.

The [AVS Device SDK 3.x documentation](https://developer.amazon.com/en-US/docs/alexa/avs-device-sdk/console-sample-app.html) states that AVS will run on Linux and MacOS as well as on a Raspberry Pi.  This issue exists to demonstrate a working example of operating one of our supported products, e.g. XVF3610, on Linux or MacOS (preferrably both).

**Comment by mbanth** [comment]
@lucianomartin, please provide a Story Point value for this issue.  Thanks.

**Comment by lucianomartin** [comment]
I tested the operation of the AVS device SDK on MAC.
I successfully installed and tested 'tap-to-talk' with the built-in mics on both Catalina and Monterey.
The latest version of the avs-device-sdk, v3.0.0, only supports Monterey, so on Catalina I installed the SDK version v1.26.0. 

I follow all the steps in the following pages:
 - v1.26.0: https://developer.amazon.com/en-US/docs/alexa/avs-device-sdk-1-2x/mac-os.html
 - v3.0.0: https://developer.amazon.com/en-US/docs/alexa/avs-device-sdk/mac-os.html
 
 The only issue I had is that my default 'python' was  not pointing to the version expected by the SDK, namely Python 2.7.x for v1.26.0 and Python 3.x.x in v3.0.0 . To address this issue I had to overwrite the 'python' command with an alias.
